packages/components/src/Illustration/subcomponents/VideoPlayer/VideoPlayer.tsx

```diff
@@ -149,6 +150,24 @@ export const VideoPlayer = ({
     }
   }, [videoRef])
 
+  useEffect(() => {
+    /**
+     * This seems counter-intuitive, but webm support is codec specific.
+     * Only offer webm if we are positive the browser supports it.
+     * Reference: https://bugs.webkit.org/show_bug.cgi?id=216652#c1
+     */
+    canPlayWebm() ? (
+      setSourceEl(
+        <>
+          <source src={assetUrl(`${source}.webm`)} type="video/webm" />
+          <source src={assetUrl(`${source}.mp4`)} type="video/mp4" />
```

Comment:
This is meant to have both the webm and mp4 versions?

Reply:
This is what the original would have had from the original code and you can have [multiple sources](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#usage_notes) in the video element.

As such I've replicated the functionality of the original implementation - but since we are checking for webm support, it does seem redundant (but maybe we want a fallback anyway?)

Reply:
Oh right, my bad I read the old code as an if else. All good 👍